README.md

```diff
@@ -96,7 +96,7 @@ If you are experiencing issues with *tldr*, consider deleting the cache files be
 # bash
 tldr --print-completion bash | sudo tee "$BASH_COMPLETION_COMPAT_DIR"/tldr
 # zsh
-tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr
+tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```

Comment:
So from the table concept, I think a better way may be something like:

```suggestion
# zsh (recommend double checking where zsh/site-functions directory is located)
# for macOS:
tldr --print-completion zsh | sudo tee /usr/local/share/zsh/site-functions/_tldr

# for Linux:
tldr --print-completion zsh | sudo tee /usr/share/zsh/site-functions/_tldr
```

Reply:
It will work however its not a fit all solution.

Reply:
Well, make whatever change you think is right, I'm done offering suggestions.